### PR TITLE
Add new option to silent approach page

### PR DESCRIPTION
**Pull request by jaspinall**
Please review this, @casey-collab.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2579)

### CODE REVIEW

File: english/explore-outside/talk-child/silent-approach/run-away/run-away.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+You realize that this demon-child means business. You turn around and flee,
```
Comment (casey-collab) [inline english/explore-outside/talk-child/silent-approach/run-away/run-away.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jaspinall** [comment]
Please review this, @casey-collab.

File: english/explore-outside/talk-child/silent-approach/silent-approach.md [hunk 2]
```diff
@@ -5,4 +5,6 @@ that is around you.
 
 You have now reached the child. What do you do?
 
-[Talk to the child anyway](../talk-more/talk-more.md)
\ No newline at end of file
+[Talk to the child anyway](../talk-more/talk-more.md)
+
+[Run away!](../run-away/run-away.md)
```
Comment (casey-collab) [inline english/explore-outside/talk-child/silent-approach/silent-approach.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/explore-outside/talk-child/silent-approach/run-away/run-away.md (hunk 1 above)
Comment (casey-collab) [inline english/explore-outside/talk-child/silent-approach/run-away/run-away.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**